packages/select/src/common/listItemsProps.ts

```diff
@@ -170,7 +170,7 @@ export interface IListItemsProps<T> extends Props {
      * created, either by pressing the `Enter` key or by clicking on the "Create
      * Item" option. It transforms a query string into an item type.
```

Comment:
I think you should add a bit to the JSDoc here about creating multiple items by returning an array.